tests/README.md

```diff
@@ -11,8 +11,7 @@ yamcs/quickstart and openmct-yamcs (this one).
 3. `make all` in yamcs/quickstart
 4. `cd openmct-yamcs` to move out of yamcs/quickstart
 5. `npm install` in openmct-yamcs
-6. `npx playwright@1.45.0 install chromium` in openmct-yamcs
```

Comment:
@unlikelyzero can you confirm that this is ok to remove?

Reply:
@ozyx before we merge this, let's check